source/content/terminus/02-install.md

```diff
@@ -37,7 +37,7 @@ Terminus does not work with the following platforms:
 </Accordion>
 
 **Terminus requires the following:**
-- PHP Version 7.4 or later
+- PHP Version 8.2 or later
```

Comment:
@jazzsequence is going to make edits to this page to make the compatibility combinations clearer

Reply:
I think this is probably no longer necessary.